### ISSUE TITLE
Distributed backups

### ISSUE THREAD

**Issue by rugk** [borgbackup/borg#2428]
One an already duplicate a backup via rsync for some redundancy as explained in https://github.com/borgbackup/borg/issues/2119 and https://github.com/borgbackup/borg/issues/1967 e.g. That's good, but this issue is about something else…

It would be nice to be able to use multiple backup repos in borg to distribute the backups to prevent a single-point-of-failure. However one should not be obliged to duplicate the whole backup as there might e.g. be space-limits on some servers or storage devices, so not the whole backup can/is allowed to be stored.

So it would also mean to split the backup. However one might also have some redundancy so one can restore the whole backup with say 3 of 5 "partial" repos available.

Example:
```
Client [5 parts]  --> server1-repo [5]
                  --> server2-repo [3]
                  --> small-usb-1 [1]
                  --> small-usb-2 [1]
                  --> small-usb-3 [1]
                  --> disk-drive-1 [2]
```

In this example one could e.g. restore the backup either with access to `server1-repo` or - if something is corrupted or the backup is not available anymore - also use the local backups to restore the whole backup too.

Maybe depends on https://github.com/borgbackup/borg/issues/1697?

**Comment by jcgruenhage** [comment]
Probably out of scope, and better handled with something generic written to serve that purpose regardless of content. That already exists, in the form of https://sia.tech/, https://storj.io/ and some more, with sia being the most promising (IMO).

**Comment by ThomasWaldmann** [comment]
There is also some overlap of this ticket with #225.

**Comment by mannp** [comment]
@jcgruenhage are you using sia yourself, as I am and am wondering the best way to utilise it with borg....?

**Comment by jcgruenhage** [comment]
@mannp no, I am not, because I don't need it. I have multiple servers to back up to ^^

**Comment by ThomasWaldmann** [comment]
borg2 maybe could use `borgstore`'s `MStore` for this.

Update: `MStore` was removed from `borgstore` master branch to start simple(r).
